app/page.tsx

```diff
@@ -1,31 +1,43 @@
 import { types } from "@mozilla/nimbus-shared";
-import { BranchInfo, RecipeOrBranchInfo, experimentColumns, FxMSMessageInfo, fxmsMessageColumns } from "./columns";
+import { RecipeOrBranchInfo, experimentColumns, FxMSMessageInfo, fxmsMessageColumns } from "./columns";
+import { getCTRPercent } from "@/lib/looker.ts";
 import { getDashboard, getDisplayNameForTemplate, getTemplateFromMessage, _isAboutWelcomeTemplate, getPreviewLink } from "../lib/messageUtils.ts";
 import { NimbusRecipeCollection } from "../lib/nimbusRecipeCollection"
 import { _substituteLocalizations } from "../lib/experimentUtils.ts";
 
 import { InfoTooltip } from "@/components/ui/infotooltip.tsx";
 import { NimbusRecipe } from "../lib/nimbusRecipe.ts"
 import { MessageTable } from "./message-table";
-import Link from "next/link";
 
 import { MenuButton } from "@/components/ui/menubutton.tsx";
 
-function getASRouterLocalColumnFromJSON(messageDef: any) : FxMSMessageInfo {
+const isLookerEnabled = process.env.IS_LOOKER_ENABLED === "true";
+
+async function getASRouterLocalColumnFromJSON(messageDef: any) : Promise<FxMSMessageInfo> {
   let fxmsMsgInfo : FxMSMessageInfo = {
     product: 'Desktop',
     id: messageDef.id,
     template: messageDef.template,
     surface: getDisplayNameForTemplate(getTemplateFromMessage(messageDef)),
     segment: 'some segment',
     metrics: 'some metrics',
-    ctrPercent: .5, // getMeFromLooker
-    ctrPercentChange: 2, // getMeFromLooker
+    ctrPercent: undefined, // getMeFromLooker
+    ctrPercentChange: undefined, // getMeFromLooker
```

Comment:
Please change the `// getMeFromLooker` in these lines to `// may be populated from Looker data`.